proposals/NNNN-GetInputScreen.md

```diff
@@ -0,0 +1,140 @@
+# Get Input Screen
+
+* Author: [Michael Crimando](https://github.com/MichaelCrimando)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / Web / RPC]
```

Comment:
Can you specify which `Web` platforms are impacted here?  Possibly SHAID, Server, and/or JavaScript Suite?